frontend/src/components/header/header.tsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 
+import { useGetWishlistsQuery } from '@store/queries/preferences/wishlist';
+
 import styles from './header.module.scss';
 import { Logo } from './logo/logo';
 import { Navigate } from './navigate/navigate';
 import { PrivateElements } from './private-elements/private-elements';
 import { PublicElements } from './public-elements/public-elements';
 
 export const Header: React.FC = () => {
+  const { data: wishlist = 0 } = useGetWishlistsQuery();
+
   const user = {
-    favorites: 3,
+    favorites: wishlist && wishlist.count,
```

Comment:
We shouldn't use `smth &&` check in the modern JS. There a optional [chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) in JS to work with possibly `undefined` objects